database/mssql/scripts/versions/v_7_ddl.sql

```diff
@@ -0,0 +1,66 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+CREATE TABLE [permit].[ORBC_TRANSACTION](
+	[TRANSACTION_ID] [bigint] IDENTITY(1,1) NOT NULL,
+	[MERCHANT_ID] [int] NOT NULL,
+	[TRANSACTION_TYPE] [varchar] (3) NOT NULL,
+	[TRANSACTION_ORDER_NUMBER] [varchar](30) NOT NULL,
```

Comment:
Technically this is up to 30 alphanumeric characters as per the Bambora docs, though I read in the BC Express Pay docs that "Using a trnOrderNumber parameter with more than ten characters in calls to the Bambora API will result in the order number value being truncated in the settlement file. This greatly reduces the usefulness of the settlement file for Transaction Verification, Reconciliation, and Remediation." I'd recommend making this 10 characters to reinforce that.

Reply:
I agree. I will update it to 10 characters.